src/plugins/intel_cpu/src/nodes/kernels/x64/jit_kernel_base.cpp

```diff
@@ -93,6 +97,19 @@ void JitKernelBase::uni_vpaddd(const Xbyak::Ymm& vDst,
     }
 }
 
+void JitKernelBase::uni_vpaddq(const Xbyak::Xmm& vDst,
+                               const Xbyak::Xmm& vSrc,
+                               const Xbyak::Operand& op) {
+    if (isValidIsa(x64::avx2)) {
```

Comment:
Seems like `avx` [supports](https://www.felixcloutier.com/x86/paddb:paddw:paddd:paddq) this instruction as well, doesn't it? Could you verify please?

Reply:
This instruction works with Xmm vectors only on the AVX arch and requires implementation like [this](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/kernels/x64/jit_kernel_base.cpp#L70). Taking into account the fact that we don't support AVX, it's better to avoid it here.